dd-sdk-android/src/main/kotlin/com/datadog/android/core/configuration/Configuration.kt

```diff
@@ -467,7 +467,7 @@ internal constructor(
             "^(([0-9]|[1-9][0-9]|1[0-9]{2}|2[0-4][0-9]|25[0-5])\\.)" +
                 "{3}([0-9]|[1-9][0-9]|1[0-9]{2}|2[0-4][0-9]|25[0-5])$|" +
                 "^(([a-zA-Z0-9]|[a-zA-Z0-9][a-zA-Z0-9\\-]*[a-zA-Z0-9])\\.)" +
-                "*([A-Za-z]|[A-Za-z][A-Za-z0-9-]*[A-Za-z0-9])$"
+                "+([A-Za-z]|[A-Za-z][A-Za-z0-9-]*[A-Za-z0-9])$"
```

Comment:
While you're there, maybe you could make this a bit more readable. Among other things, the cardinality modifiers should be at the end of a line, not the beginning. E.g.: `"(…){3}" + "…"` or  `"(…)\\.+" + "(…)$"`